doc/specs/vulkan/chapters/fundamentals.txt

```diff
@@ -707,13 +726,21 @@ commands, which will silently ignore these values.
 [[fundamentals-validusage-pointers]]
 ==== Valid Usage for Pointers
 
-Any parameter that is a pointer must: either be a valid pointer, or if
-_explicitly called out in the specification_, can: be `NULL`.
 A pointer is valid if it points at memory containing values of the number
 and type(s) expected by the command, and all fundamental types accessed
 through the pointer (e.g. as elements of an array or as members of a
 structure) satisfy the alignment requirements of the host processor.
 
+Valid Usage section of a command or of a relevant parameter type does state
+if a command requires a valid pointer, or not.
+
+[NOTE]
+.Note
+====
+To some commands under some conditions a pointer can: be passed that is not
```

Comment:
"Under some conditions, a pointer can: be specified to command parameters or structure members that is NULL, or is not valid as defined above. In such cases the pointer value is ignored."

Reply:
That would be a lie. `NULL` value is likely not ignored.
Then again, the application should not really care as long as it is not dereferenced.
I guess I will try something new.
What I wanted to do is to inform that some command descriptions say a parameter is "ignored". Then I wanted to say that lack of "valid" in the VU means it does not have to be valid (which was implied by above specifying sentence, but not said explicitly/humanely)

Reply:
How about:

"Pointer parameters passed to commands can additionally be `NULL` when specified by Valid Usage. If no valid usage is specified for a given pointer in a given circumstance, it can have any value as the parameter will be ignored by the calling command."

Reply:
@TobiasHector  I should not overthink it; it is only a Note, but:
- it cannot be "additionally"(to valid values) `NULL` in all cases. Sometimes it is "conditionally" `NULL`. Sometimes "only" `NULL` (mostly `pNext` with no known extensions).
- sometimes a valid usage is specified for a pointer parameter, but it still can be invalid (e.g. above in the `vkMapMemory`. Only the lack of word "valid" is significant

Reply:
...can we live without the note :P?

Reply:
@TobiasHector We shall see. I guess first I should export the important stuff to other PRs, because on this there probably will be endless controversy and endless word-smithery despite relatively low added value :P

Reply:
@TobiasHector Alright. Trying to live without it. 💔